src/core/index.js

```diff
@@ -37,31 +37,3 @@ exports._test = {
 exports.sjcl = require('./utils').sjcl;
 exports.Wallet = require('ripple-wallet-generator')({sjcl: exports.sjcl});
 exports.types = require('./serializedtypes');
-
-// camelCase to under_scored API conversion
-function attachUnderscored(name) {
```

Comment:
Why remove this? This will cause breakage beyond needing to fix the require to dereference _DEPRECATED such that `deprecating core` doesn't fully describe the change ?  The `core` will now be internals that you will be free to change the implementation of, at will ?

> // WARNING: this will be removed soon`

How soon? What's the plan here? (Sorry if this was mentioned on architects. I wasn't really paying attention)